requirements.txt

```diff
@@ -14,7 +14,5 @@ torch==1.1.0
 torchvision==0.3.0
 tensorboard==1.14.0
 tensorboardX==1.8
-gitpython==3.1.0
-tensorflow==1.15.2
```

Comment:
Should remove any other used requirements

Reply:
should there be a python requirement in here?